### PR TITLE
Song player observes the song loader and is also observable

### PR DESCRIPTION
**Pull request by Ben-Graham**

### CODE REVIEW

File: tags [hunk 1]
```diff
@@ -0,0 +1,166 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
```
Comment (bseto) [inline tags:1]: You should remove this tags file from the repository

**Comment by bseto** [comment]
So Ben. You need to make sure the pull request can compile, and you need to remove the tags file.

**Comment by ExtraGravity** [comment]
Has the `.travis.yml` been updated? If not, please do so in this PR. Also, consider adding tags to the gitignore so you don't have to worry about that next time.

**Comment by bseto** [comment]
This will need to rebase/merge from master and then have tests added. Closing pull request for now.

**Pull request closed without merging**